pkg/controller/node/status.go

```diff
@@ -89,6 +89,10 @@ func calculateStatus(pool *mcfgv1.MachineConfigPool, nodes []*corev1.Node) mcfgv
 		mcfgv1.SetMachineConfigPoolCondition(&status, *supdated)
 		supdating := mcfgv1.NewMachineConfigPoolCondition(mcfgv1.MachineConfigPoolUpdating, corev1.ConditionTrue, "", fmt.Sprintf("All nodes are updating to %s", pool.Spec.Configuration.Name))
 		mcfgv1.SetMachineConfigPoolCondition(&status, *supdating)
+
+		if pool.Spec.Paused == true { // if the pool is Paused and updating, the pool will never progress.
```

Comment:
Q: as per https://github.com/openshift/machine-config-operator/blob/fe05cca6e8784f60dfc03faa9684672b30e24711/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L222

Is this the correct place for this? 

Or is https://github.com/openshift/machine-config-operator/blob/cd925586f71d709a36a11945a7abf8104e3d2046/pkg/controller/node/node_controller.go#L723 better?